internal/e2e/rollup/BUILD.bazel

```diff
@@ -3,11 +3,8 @@ load("//:defs.bzl", "rollup_bundle")
 
 rollup_bundle(
     name = "bundle",
-    srcs = [
-        "bar.js",
-        "foo.js",
-    ],
-    entry_point = "internal/e2e/rollup/foo.js",
+    srcs = ["bar.js"],
+    entry_point = ":foo.js",
```

Comment:
interesting that you don't have to specify foo.js in the srcs. It seems like it reduces one way users can fall off the happy path. on the other hand it means you have to consider entry_point when enumerating the inputs to a rule.
Should we give it more thought?

Reply:
Hmmm. That's a good question. One strike against doing it this way is that if the user specifies `:foo.ts` as the entry_point they still have to add the ts_library `:foo_lib` to deps. I'm not sure what is best. On the other hand, the user adding `:foo.js` to entry_point and to srcs doesn't break anything with the current approach.

Reply:
There is some smell however with not having the file in `srcs` as seen here: https://github.com/bazelbuild/rules_nodejs/pull/777/files#diff-6e8a939ed5da5eca956cd2f53701b77e. Probably better to force it to be specified